algorithms/CPlusPlus/Stacks/stock-span-problem.cpp

```diff
@@ -0,0 +1,60 @@
+
+// C++ linear time solution for stock span problem
+#include <iostream>
+#include <stack>
+using namespace std;
+ 
+// A stack based efficient method to calculate
+// stock span values
+void calculateSpan(int price[], int n, int S[])
```

Comment:
If `n` holds the value of size of the array `price []`, and is only being treated as a read-only variable thought the function body then shouldn't it be a good idea to make it `const` ?

Reply:
void calculateSpan(const vector<int> price, const int n, int S[])